C/cargo_license/build_tarballs.jl

```diff
@@ -0,0 +1,46 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "cargo_license"
+version = v"0.4.2"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/onur/cargo-license.git", "f489e18b0fc45ab31d9766841e7f551e96ee6a8c"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+
+
+cd $WORKSPACE/srcdir/cargo-license
+
+cargo build --release --all
+
+mkdir ${bindir}
+cp target/${rust_target}/release/cargo-license${exeext} ${bindir}/
+
+cargo-license --avoid-build-deps --avoid-dev-deps --json > CARGO_LICENSES.json
```

Comment:
Am I missing something or this is going to fail again?

Reply:
This is the 'bootstrap' call of cargo-license on the cargo-license library I build with this script

Reply:
But it does fail for other reasons. ;)

Reply:
five minutes later....You aren't missing anything.